src/openzeppelin/tests/test_accesscontrol.cairo

```diff
@@ -60,7 +63,7 @@ fn test_has_role() {
 fn test_assert_only_role() {
     setup();
     AccessControl::grant_role(ROLE, AUTHORIZED());
-    set_caller_address(AUTHORIZED());
+    testing::set_caller_address(AUTHORIZED());
```

Comment:
```suggestion
    testing::set_caller_address(AUTHORIZED);
```
we can now change these functions for top level constants, no?

Reply:
Unfortunately, no. In the Account PR, the public key was a felt252 (which I think this is in reference to), so that can be a constant. No support yet for contract addresses though